features/networking/ib-sriov.feature

```diff
@@ -0,0 +1,158 @@
+Feature: Sriov IB related scenarios
+
+  # @author zzhao@redhat.com
```

Comment:
Need a case_id here

Reply:
this is `Outline` case, you can see the case id in line 63-65

Reply:
Get it, thanks!